README.md

```diff
@@ -41,37 +45,61 @@
 		
 </table>
 
-### How to Run Locally
-Run the below command to clone the repo locally
+## How to Contribute:
+- <b>Fork the Project Repository :</b><br>
+You can make a copy of the project to your account in order to make contribution.This process is called forking a project to your Github account.Click on the fork button to fork the project.
+
+![Screenshot (247)](https://user-images.githubusercontent.com/61612939/113122940-bd2fa080-9231-11eb-9c62-61d2eb711f7c.png)
+
+- <b>Clone the Project Repository :</b><br>
+To get this project on your development machine we use clone command of git.Run the below command to clone the repo locally
 ```
 git clone https://github.com/kothariji/SyntaxMeets.git
 ``` 
 
-### To run the server, go to the server folder in the backend directory and Run 
+- <b>Add a Remote the Original Project Repository: </b><br>
+Remote means the remote location of project on Github.This will point to the original repository. It will help you to get the latest changes made by others.
+```
+$ cd <your-forked-project-folder>
+$ git remote add upstream https://github.com/<author-account-username>/<project>.git
+``` 
+
+- <b>To run the server, go to the server folder in the backend directory and Run: </b><br>
 ```
 npm install
 npm start
 ```
-#### this will run your express server in localhost:4000
+<b>This will run your express server in localhost:4000</b>
 
 ---
-### To run the application, go to the SyntaxMeets folder in the frontend directory 
+- <b>To run the application, go to the SyntaxMeets folder in the frontend directory. Create a new .env file and add following variables:</b>
 
-create a new .env file and add following variables
 ```
 REACT_APP_SYNTAXMEETS_BACKEND_API = 'http://localhost:4000/'
 REACT_APP_ONLINE_JUDGE_API = YOUR API KEY
 ```
 
-and Run the application using below commands
+- <b>Run the application using below commands</b>
 ```
 npm install		//install all react dependencies
 npm start		//runserver
 ```
-#### this will run your application in localhost:3000
+<b>This will run your application in localhost:3000</b>
+
+## Contribution Guidelines:<br>
+-To contribute to this project, go through the step-by-step guidelines provided [here.](https://github.com/kothariji/SyntaxMeets/blob/master/CONTRIBUTING.md)
+Contributors should strictly abide by the following points while contributing:<br>
+- Make sure you do not copy code from external sources because that work will not be considered.<br>
+- Plagiarism is strictly not allowed.<br>
```

Comment:
Can you please explain what do you mean by plagiarism here  ? As this repository is not the collection of algorithms or similar thing, where plagiarism matters.

Reply:
With plagiarism, it is meant to say that while contributing, one should not copy the ideas or the code of any other project similar to this project. There are many other projects with similar idealogy with this project therefore we should have some uniqueness which would differentiate them so, to make sure that this uniqueness is maintained it is necessary that while contributing one should make sure that the ideas proposed by him/her are unique and their own and not copied from any other source.

Reply:
Cool! I agree with this point 🙌